### PR TITLE
Testing v2

### PR DESCRIPTION
**Pull request by ohadsahar**

### CODE REVIEW

File: __tests__/ProgressLine.test.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import ProgressLine from "@/core/components/ProgressLine/progress-line";
+import "@testing-library/jest-dom/extend-expect";
+import { render, screen } from "@testing-library/react";
+import AppTestUtil from "../app_test.util";
+
+describe("Checking Layout component", () => {
+  it("Testing Layout component exists on DOM", () => {
+    render(
+      <AppTestUtil>
+        <ProgressLine></ProgressLine>
```
Comment (amitmarko) [inline __tests__/ProgressLine.test.tsx:10]:
```suggestion
        <ProgressLine />
```

File: core/components/SearchBar/SearchBar.tsx [hunk 2]
```diff
@@ -7,7 +7,10 @@ const SearchBar = () => {
   const { countSearchedTasks, onChange } = useSearchBar();
 
   return (
-    <FormWrapper onSubmit={() => event?.preventDefault()}>
+    <FormWrapper
+      onSubmit={(event) => event?.preventDefault()}
```
Comment (amitmarko) [inline core/components/SearchBar/SearchBar.tsx]:
```suggestion
      onSubmit={(e: React.SyntheticEvent) => e.preventDefault()}
```

File: store/actions/alert.actions.ts [hunk 3]
```diff
@@ -1,9 +1,7 @@
-import { Dispatch } from "redux";
-import { v4 as uuid } from "uuid";
 import { REMOVE_ALERT, SET_ALERT } from "../types/alert.types";
 
 export const setAlert = (msg: string, alertType: string) => (dispatch: any) => {
-  const id = uuid();
+  const id = (Math.floor(Math.random() * 100) + Date.now()).toString();
```
Comment (amitmarko) [inline store/actions/alert.actions.ts]: Create generateId function 

```suggestion
  const id = generateId();
```

File: store/actions/tasks.actions.tsx [hunk 4]
```diff
@@ -5,15 +5,14 @@ import store from "@/store/store";
...
-import { v4 as uuid } from "uuid";
 import { setAlert } from "./alert.actions";
 
 export const loadTasks = () => (dispatch: any) => {
   dispatch(resetDataFromLocalStorage(SET_TASK));
 };
 
 export const createTask = (taskName: string) => (dispatch: any) => {
-  const id = uuid();
+  const id = (Math.floor(Math.random() * 100) + Date.now()).toString();
```
Comment (amitmarko) [inline store/actions/tasks.actions.tsx]:
```suggestion
  const id = generateId();
```

File: core/components/SearchBar/SearchBar.tsx (hunk 2 above)
Comment (ohadsahar) [reply core/components/SearchBar/SearchBar.tsx]: Done

Comment (amitmarko) [reply core/components/SearchBar/SearchBar.tsx]: update line 17 as well:
`onChange={(e: ...) => onChange(e.target.value)}`

File: store/actions/alert.actions.ts (hunk 3 above)
Comment (ohadsahar) [reply store/actions/alert.actions.ts]: Created a service for this

File: store/actions/tasks.actions.tsx (hunk 4 above)
Comment (ohadsahar) [reply store/actions/tasks.actions.tsx]: Same

File: core/components/SearchBar/SearchBar.tsx [hunk 5]
```diff
@@ -1,13 +1,17 @@
...
 import { useSearchBar } from "./hooks/useSearchBar";
 import { FormWrapper, InputFieldWrapper, InputWrapper } from "./styled";
 
 const SearchBar = () => {
   const { countSearchedTasks, onChange } = useSearchBar();
 
   return (
-    <FormWrapper onSubmit={() => event?.preventDefault()}>
+    <FormWrapper
+      onSubmit={(event: React.SyntheticEvent) => event?.preventDefault()}
```
Comment (amitmarko) [inline core/components/SearchBar/SearchBar.tsx]: no need to check if event have value

```suggestion
      onSubmit={(event: React.SyntheticEvent) => event.preventDefault()}
```

**Review by amitmarko — APPROVED** [review]
LGTM

**Review by amitmarko — APPROVED** [review]

**Pull request merged**